src/js/components/curriculum/CurriculumTable.js

```diff
@@ -20,9 +20,13 @@ export class CurriculumTable extends React.Component {
   }
 
   loadCurriculumContent() {
-    const { hasBeenFetched, curriculumList } = this.props.curriculum;
+    let { hasBeenFetched, curriculumList } = this.props.curriculum;
     if (!hasBeenFetched) return <Loader owner="curriculum"/>;
     if (curriculumList.length) {
+      // sort curriculumList based on order
+      curriculumList = curriculumList.sort((objA, objB) => {
+        return objA.order - objB.order;
+      })
```

Comment:
Since the sort callback is a one liner, you could do away with the `return` statement. Line 27 - 29 could be
 `curriculumList = curriculumList.sort((objA, objB) => objA.order - objB.order )`